com.unity.shadergraph/Documentation~/Refract-Node.md

```diff
@@ -0,0 +1,42 @@
+# Refract Node
+
+## Description
+
+Generates a refraction [Ibn Sahl-Snell-Descartes's Law](https://en.wikipedia.org/wiki/Snell%27s_law). Based on an **Incident** vector, the **Normal** of the surface and **Eta** we produce the refracted vector.
```

Comment:
Are all of the names really necessary, if it's more commonly known as just "Snell's law"? Even the link you're pointing to is just "Snell's Law". I can see this being unnecessarily confusing. 

> Generates a refraction using [Snell's Law](https://en.wikipedia.org/wiki/Snell%27s_law) and produces a new refracted vector.

Reply:
It's Snell's Law only on anglo-saxon world, I learnt this as "Descartes's Law".
I'll push an simpler update.
Wikipedia:
"The law eventually named after Snell was first accurately described by the Persian scientist Ibn Sahl at the Baghdad court in 984"
It's to be fair, Sahl Law is 600 years before Snell.
Ref:
https://twitter.com/wkjarosz/status/1443562779701547012?s=20